packages/vx-voronoi/src/components/VoronoiPolygon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cx from 'classnames';
+
+export type VoronoiPolygonProps = {
+  /** Override render function which is provided polygon and generated path. */
+  children?: ({ path, polygon }: { path: string; polygon: number[][] }) => React.ReactNode;
+  /** className to apply to path element. */
+  className?: string;
+  /** Array of coordinate arrays for the polygon (e.g., [[x,y], [x1,y1], ...]), used to generate polygon path. */
+  polygon?: number[][];
```

Comment:
`[number, number][]`

Reply:
looks like this is not possible when the type is optional, build fails with [this error](https://github.com/Microsoft/TypeScript/issues/12707#issuecomment-384878950)

Reply:
made it required to avoid

Reply:
I take that back, I think it comes from the [`@types/d3-voronoi` types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-voronoi/index.d.ts#L177)